http/http.go

```diff
@@ -72,10 +66,15 @@ func Conflict(w http.ResponseWriter, err error) {
 }
 
 func HTTPError(w http.ResponseWriter, code int, err error) {
+	logger := logs.WithTag("code", code)
 	if err != nil {
-		logs.WithTag("code", code).
-			WithTag("error", err.Error()).
-			Warn("http request returned an error")
+		logger = logger.WithTag("error", err)
```

Comment:
Please move this to the non-500 section, and in the 500-or-above section wrap the original error.